README.md

```diff
@@ -79,22 +79,22 @@
 
 <details>
 <summary>Screen Shots</summary>
-<img src="./screenshots/ss-1.png"/>
-<img src="./screenshots/ss-2.png"/>
-<img src="./screenshots/ss-3.png"/>
-<img src="./screenshots/ss-4.png"/>
-<img src="./screenshots/ss-5.png"/>
+<img src="./screenshots/1.png"/>
+<img src="./screenshots/2.png"/>
+<img src="./screenshots/3.png"/>
+<img src="./screenshots/4.png"/>
+<img src="./screenshots/5.png"/>
 </details>
 
 <!-- Video -->
 ## :movie_camera: Video documentation
-[video](https://drive.google.com/file/d/1J3jDFveZznVjx5uZs9z9lN_yW7bPMWUU/view?usp=sharing)
+[video](https://www.loom.com/share/e33aa6cdacb3432bacf19edfc2487d9c)
```

Comment:
- [ ] [REQUIRED] Great effort you have put into the video presentation, unfortunately, there seems to be some problem with your microphone as audio was not included in the presentation. Kindly record another video and do not forget to highlight a piece of the code that you are proud of making. By doing this, you will be adhering fully to the project requirements.
- [ ] [OPTIONAL] I will recommend that you record the mobile view of your application as this project centers on mobile view. It is also good to show the viewers the beauty of the desktop interface as well.
- You have a great user interface and experience 🥇 💪🏽